lib/rubycritic/configuration.rb

```diff
@@ -2,7 +2,8 @@ module Rubycritic
   class Configuration
     attr_reader :root
     attr_accessor :source_control_system, :mode, :format, :deduplicate_symlinks,
-                  :suppress_ratings, :open_with, :no_browser
+                  :suppress_ratings, :open_with, :no_browser, :assets_folder,
```

Comment:
Same here with cohesion and coupling - all configuration values here are completely generic but all of a sudden there are 4 highly rails specific configuration values in here. This should be a separate configuration object which is then included into this one via composition.